shuup/reporting/reports/sales.py

```diff
@@ -0,0 +1,126 @@
+# -- encoding: UTF-8 --
+import itertools
+
+import six
+from django.db.models import Sum
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.core.models import Order
+from shuup.core.pricing import TaxfulPrice, TaxlessPrice
+from shuup.reporting.reporter.base import ShuupReportBase
+
+
+class OrderReportMixin(object):
+    def get_objects(self):
+        return Order.objects.filter(
```

Comment:
This should probably filter cancelled orders or there should be option to remove cancelled orders from the report.